projects/plugins/wpcomsh/woa.php

```diff
@@ -393,3 +393,65 @@ function wpcomsh_woa_post_process_store_woocommerce_connection_details( $args, $
 add_action( 'wpcomsh_woa_post_clone', 'wpcomsh_woa_post_process_store_woocommerce_connection_details', 10, 2 );
 add_action( 'wpcomsh_woa_post_reset', 'wpcomsh_woa_post_process_store_woocommerce_connection_details', 10, 2 );
 add_action( 'wpcomsh_woa_post_transfer', 'wpcomsh_woa_post_process_store_woocommerce_connection_details', 10, 2 );
+
+/**
+ * Ensures that specific Jetpack modules are activated after a transfer.
+ * Addresses the issue where certain modules like blocks, account-protection and blaze
+ * may be disabled during the transfer process.
+ *
+ * @param array $args       Arguments.
+ * @param array $assoc_args Associated arguments.
+ */
+function wpcomsh_woa_post_process_activate_jetpack_modules( $args, $assoc_args ) {
+
+	if ( ! is_plugin_active( 'jetpack/jetpack.php' ) ) {
+		WP_CLI::warning( 'Jetpack plugin is not active, skipping module activation' );
+		return;
+	}
+
+	// First, make sure the jetpack_blocks_disabled option is deleted
+	delete_option( 'jetpack_blocks_disabled' );
```

Comment:
Do we need to delete that option when it's done later when the module gets activated?
https://github.com/Automattic/jetpack/blob/9c556174eee294544a57d736d095ef05c8539d8e/projects/plugins/jetpack/modules/blocks.php#L15-L30